.gitignore

```diff
@@ -1,6 +1,8 @@
+.DS_Store
```

Comment:
Why not? Doesn't make sense since we are working with macs? Otherwise I always have to remove and keep and eye on them